system/include/wasm_simd128.h

```diff
@@ -1,6 +1,8 @@
 /*
 WebAssembly SIMD128 Intrinsics
 */
+#ifndef _WASM_SIMD128_
+#define _WASM_SIMD128_
```

Comment:
Maybe make this `_WASM_SIMD128_H_` to reduce the probability of this conflicting with some other macro by accident.